lhotse/recipes/audio_mnist.py

```diff
@@ -0,0 +1,110 @@
+"""
+About the AudioMNIST corpus
+
+The AudioMNIST dataset consists of 30000 audio recordings
+(ca. 9.5 hours) of spoken digits (0-9) in English with 50 repetitions
+per digit for each of the 60 different speakers. Recordings were
+collected in quiet offices with a RØDE NT-USB microphone as
+mono channel signal at a sampling frequency of 48kHz and were
+saved in 16 bit integer format. In addition to audio recordings, meta
+information including age (range: 22 - 61 years), gender (12 female
+/ 48 male), origin and accent of all speakers were collected as well.
+All speakers were informed about the intent of the data collection
+and have given written declarations of consent for their participation prior
+to their recording session.
+
+We don't provide download_audio_mnist().
```

Comment:
technically we could provide it and let it call `subprocess.run("git clone ...", shell=True)`, WDYT?

Reply:
github produces tarballs automatically: https://github.com/soerenab/AudioMNIST/archive/master.tar.gz

Reply:
> github produces tarballs automatically: https://github.com/soerenab/AudioMNIST/archive/master.tar.gz

Thanks! Will add the download command.

Reply:
Should I just merge it? We can always add download later if needed.

Reply:
Sorry for the late reply. I just added it.